### DISCUSSION TITLE
The new feed won't show the activities of the people I follow

### DISCUSSION THREAD

**Discussion by parksb** [community/community#67091] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I used to check the activities of the people I follow in the old "Following" feed. I could get helpful information from the feed about the repositories that were starred or created by other people. However, the new feed does not provide any of these features.

![image](https://github.com/community/community/assets/6410412/fd606030-746c-4e0e-acc1-9c75dc55dabb)

No matter what filter options I apply in the new feed, I can't be provided activities of the people I follow in the feed. It just shows other people's activities on "my repository". Is it a feature or a bug?

**Comment by mahimairaja** [comment]
It is the feature, but I could see many people are getting annoyed by this update though.. LOL

**Comment by xieyuheng** [comment]
github is going bad.
